collection-scripts/gather_network_logs

```diff
@@ -23,6 +23,24 @@ for NODE in $NODES; do
     PIDS+=($!)
     oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
     PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "ip a" > ${NETWORK_LOG_PATH}/${NODE}_ip &
+    PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "brctl show" > ${NETWORK_LOG_PATH}/${NODE}_brctl &
```

Comment:
The fact that `brctl` is still present in the openshift-sdn image is an oversight that will probably be fixed in the future. (OpenShift SDN doesn't use linux bridges and doesn't care if anyone else does.) Don't use the `openshift-sdn` pod to collect non-openshift-sdn-related information.